ACMAS/app/acmas_tables.sql

```diff
@@ -0,0 +1,60 @@
+create table "ACMAS_Web_university"
+(
+    id   bigserial
+        primary key,
+    name varchar(50) not null
+);
+
+alter table "ACMAS_Web_university"
+    owner to hello_acmas;
+
+create table "ACMAS_Web_course"
+(
+    id         bigserial
+        primary key,
+    name       text        not null,
+    code       varchar(20) not null,
+    university text        not null,
+    semster    varchar(30) not null,
+    years      text        not null,
+    test_type  varchar(20) not null
+);
```

Comment:
This is probably for search engine PR, but we would like to add a column called `question_ids` for each exam/hw item. The `question_ids` link to the IDs in "ACMAS_Web_question" table included for the exam/hw. It would be a list of bigserial ids from "ACMAS_Web_question".

Reply:
I will implement it once you guys are ready! I can also implement it in my typo-fixing issue/PR that will be opened after this one.